<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="3" total="45" passed="28" failed="14" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-08-03T15:14:59 IST" name="Suite" finished-at="2023-08-03T15:21:52 IST" duration-ms="412779">
    <groups>
    </groups>
    <test started-at="2023-08-03T15:14:59 IST" name="Test" finished-at="2023-08-03T15:21:52 IST" duration-ms="412779">
      <class name="tests.LoginTest">
        <test-method is-config="true" signature="init()[pri:0, instance:tests.LoginTest@5d0a1059]" started-at="2023-08-03T15:14:59 IST" name="init" finished-at="2023-08-03T15:14:59 IST" duration-ms="321" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.LoginTest@5d0a1059]" started-at="2023-08-03T15:15:00 IST" name="setUp" finished-at="2023-08-03T15:15:04 IST" duration-ms="4078" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkLogin(java.lang.String,java.lang.String)[pri:0, instance:tests.LoginTest@5d0a1059]" started-at="2023-08-03T15:15:04 IST" name="checkLogin" data-provider="fetchData" finished-at="2023-08-03T15:15:05 IST" duration-ms="580" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.LoginTest@5d0a1059]" started-at="2023-08-03T15:15:05 IST" name="tearDown" finished-at="2023-08-03T15:15:06 IST" duration-ms="726" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.LoginTest@5d0a1059]" started-at="2023-08-03T15:15:06 IST" name="setUp" finished-at="2023-08-03T15:15:08 IST" duration-ms="2201" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkLogin(java.lang.String,java.lang.String)[pri:0, instance:tests.LoginTest@5d0a1059]" started-at="2023-08-03T15:15:08 IST" name="checkLogin" data-provider="fetchData" finished-at="2023-08-03T15:15:08 IST" duration-ms="480" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.LoginTest@5d0a1059]" started-at="2023-08-03T15:15:08 IST" name="tearDown" finished-at="2023-08-03T15:15:08 IST" duration-ms="220" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.LoginTest@5d0a1059]" started-at="2023-08-03T15:15:08 IST" name="setUp" finished-at="2023-08-03T15:15:11 IST" duration-ms="2363" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkLogin(java.lang.String,java.lang.String)[pri:0, instance:tests.LoginTest@5d0a1059]" started-at="2023-08-03T15:15:11 IST" name="checkLogin" data-provider="fetchData" finished-at="2023-08-03T15:15:16 IST" duration-ms="5486" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[performance_glitch_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.LoginTest@5d0a1059]" started-at="2023-08-03T15:15:16 IST" name="tearDown" finished-at="2023-08-03T15:15:17 IST" duration-ms="241" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.LoginTest@5d0a1059]" started-at="2023-08-03T15:15:17 IST" name="setUp" finished-at="2023-08-03T15:15:19 IST" duration-ms="2460" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkLogin(java.lang.String,java.lang.String)[pri:0, instance:tests.LoginTest@5d0a1059]" started-at="2023-08-03T15:15:19 IST" name="checkLogin" data-provider="fetchData" finished-at="2023-08-03T15:15:19 IST" duration-ms="365" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[locked_out_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.LoginTest@5d0a1059]" started-at="2023-08-03T15:15:19 IST" name="tearDown" finished-at="2023-08-03T15:15:20 IST" duration-ms="204" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.LoginTest -->
      <class name="tests.CheckOutTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:18:04 IST" name="setUp" finished-at="2023-08-03T15:18:07 IST" duration-ms="2499" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkOut(java.lang.String,java.lang.String)[pri:7, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:18:07 IST" name="checkOut" data-provider="fetchData" finished-at="2023-08-03T15:18:11 IST" duration-ms="4312" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkOut -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:18:11 IST" name="tearDown" finished-at="2023-08-03T15:18:11 IST" duration-ms="260" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:18:11 IST" name="setUp" finished-at="2023-08-03T15:18:14 IST" duration-ms="2506" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkOut(java.lang.String,java.lang.String)[pri:7, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:18:14 IST" name="checkOut" data-provider="fetchData" finished-at="2023-08-03T15:18:18 IST" duration-ms="4287" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at tests.CheckOutTest.checkOut(CheckOutTest.java:31)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkOut -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:18:18 IST" name="tearDown" finished-at="2023-08-03T15:18:19 IST" duration-ms="244" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:18:19 IST" name="setUp" finished-at="2023-08-03T15:18:21 IST" duration-ms="2289" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkOut(java.lang.String,java.lang.String)[pri:7, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:18:21 IST" name="checkOut" data-provider="fetchData" finished-at="2023-08-03T15:18:30 IST" duration-ms="9260" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[performance_glitch_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkOut -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:18:30 IST" name="tearDown" finished-at="2023-08-03T15:18:30 IST" duration-ms="181" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:18:30 IST" name="setUp" finished-at="2023-08-03T15:18:33 IST" duration-ms="2316" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkOut(java.lang.String,java.lang.String)[pri:7, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:18:33 IST" name="checkOut" data-provider="fetchData" finished-at="2023-08-03T15:18:43 IST" duration-ms="10766" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[locked_out_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at tests.CheckOutTest.checkOut(CheckOutTest.java:31)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkOut -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:18:43 IST" name="tearDown" finished-at="2023-08-03T15:18:44 IST" duration-ms="184" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:18:44 IST" name="setUp" finished-at="2023-08-03T15:18:46 IST" duration-ms="2616" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkTotal_Prod_WithCartItems(java.lang.String,java.lang.String)[pri:8, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:18:46 IST" name="checkTotal_Prod_WithCartItems" data-provider="fetchData" finished-at="2023-08-03T15:18:53 IST" duration-ms="6397" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTotal_Prod_WithCartItems -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:18:53 IST" name="tearDown" finished-at="2023-08-03T15:18:53 IST" duration-ms="208" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:18:53 IST" name="setUp" finished-at="2023-08-03T15:18:55 IST" duration-ms="2506" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkTotal_Prod_WithCartItems(java.lang.String,java.lang.String)[pri:8, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:18:55 IST" name="checkTotal_Prod_WithCartItems" data-provider="fetchData" finished-at="2023-08-03T15:19:02 IST" duration-ms="6226" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at tests.CheckOutTest.checkTotal_Prod_WithCartItems(CheckOutTest.java:62)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTotal_Prod_WithCartItems -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:02 IST" name="tearDown" finished-at="2023-08-03T15:19:02 IST" duration-ms="239" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:02 IST" name="setUp" finished-at="2023-08-03T15:19:04 IST" duration-ms="2354" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkTotal_Prod_WithCartItems(java.lang.String,java.lang.String)[pri:8, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:04 IST" name="checkTotal_Prod_WithCartItems" data-provider="fetchData" finished-at="2023-08-03T15:19:16 IST" duration-ms="11410" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[performance_glitch_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTotal_Prod_WithCartItems -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:16 IST" name="tearDown" finished-at="2023-08-03T15:19:16 IST" duration-ms="250" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:16 IST" name="setUp" finished-at="2023-08-03T15:19:19 IST" duration-ms="2527" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkTotal_Prod_WithCartItems(java.lang.String,java.lang.String)[pri:8, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:19 IST" name="checkTotal_Prod_WithCartItems" data-provider="fetchData" finished-at="2023-08-03T15:19:31 IST" duration-ms="12745" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[locked_out_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at tests.CheckOutTest.checkTotal_Prod_WithCartItems(CheckOutTest.java:62)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTotal_Prod_WithCartItems -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:31 IST" name="tearDown" finished-at="2023-08-03T15:19:32 IST" duration-ms="226" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:32 IST" name="setUp" finished-at="2023-08-03T15:19:34 IST" duration-ms="2386" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkFieldInformation(java.lang.String,java.lang.String)[pri:9, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:34 IST" name="checkFieldInformation" data-provider="fetchData" finished-at="2023-08-03T15:19:37 IST" duration-ms="3085" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkFieldInformation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:37 IST" name="tearDown" finished-at="2023-08-03T15:19:37 IST" duration-ms="223" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:37 IST" name="setUp" finished-at="2023-08-03T15:19:40 IST" duration-ms="2694" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkFieldInformation(java.lang.String,java.lang.String)[pri:9, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:40 IST" name="checkFieldInformation" data-provider="fetchData" finished-at="2023-08-03T15:19:43 IST" duration-ms="2989" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkFieldInformation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:43 IST" name="tearDown" finished-at="2023-08-03T15:19:43 IST" duration-ms="242" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:43 IST" name="setUp" finished-at="2023-08-03T15:19:46 IST" duration-ms="2439" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkFieldInformation(java.lang.String,java.lang.String)[pri:9, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:46 IST" name="checkFieldInformation" data-provider="fetchData" finished-at="2023-08-03T15:19:54 IST" duration-ms="8135" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[performance_glitch_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkFieldInformation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:54 IST" name="tearDown" finished-at="2023-08-03T15:19:54 IST" duration-ms="244" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:54 IST" name="setUp" finished-at="2023-08-03T15:19:57 IST" duration-ms="2476" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkFieldInformation(java.lang.String,java.lang.String)[pri:9, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:19:57 IST" name="checkFieldInformation" data-provider="fetchData" finished-at="2023-08-03T15:20:07 IST" duration-ms="10768" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[locked_out_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at tests.CheckOutTest.checkFieldInformation(CheckOutTest.java:99)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkFieldInformation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:20:08 IST" name="tearDown" finished-at="2023-08-03T15:20:08 IST" duration-ms="197" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:20:08 IST" name="setUp" finished-at="2023-08-03T15:20:10 IST" duration-ms="2448" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkOrderCompletion(java.lang.String,java.lang.String)[pri:10, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:20:10 IST" name="checkOrderCompletion" data-provider="fetchData" finished-at="2023-08-03T15:20:19 IST" duration-ms="8575" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkOrderCompletion -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:20:19 IST" name="tearDown" finished-at="2023-08-03T15:20:19 IST" duration-ms="180" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:20:19 IST" name="setUp" finished-at="2023-08-03T15:20:22 IST" duration-ms="2650" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkOrderCompletion(java.lang.String,java.lang.String)[pri:10, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:20:22 IST" name="checkOrderCompletion" data-provider="fetchData" finished-at="2023-08-03T15:20:30 IST" duration-ms="8365" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at tests.CheckOutTest.checkOrderCompletion(CheckOutTest.java:140)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkOrderCompletion -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:20:30 IST" name="tearDown" finished-at="2023-08-03T15:20:30 IST" duration-ms="186" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:20:30 IST" name="setUp" finished-at="2023-08-03T15:20:34 IST" duration-ms="3221" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkOrderCompletion(java.lang.String,java.lang.String)[pri:10, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:20:34 IST" name="checkOrderCompletion" data-provider="fetchData" finished-at="2023-08-03T15:20:47 IST" duration-ms="13550" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[performance_glitch_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkOrderCompletion -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:20:47 IST" name="tearDown" finished-at="2023-08-03T15:20:47 IST" duration-ms="222" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:20:47 IST" name="setUp" finished-at="2023-08-03T15:20:50 IST" duration-ms="2408" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkOrderCompletion(java.lang.String,java.lang.String)[pri:10, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:20:50 IST" name="checkOrderCompletion" data-provider="fetchData" finished-at="2023-08-03T15:21:03 IST" duration-ms="12764" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[locked_out_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at tests.CheckOutTest.checkOrderCompletion(CheckOutTest.java:140)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkOrderCompletion -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:21:03 IST" name="tearDown" finished-at="2023-08-03T15:21:03 IST" duration-ms="197" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:21:03 IST" name="setUp" finished-at="2023-08-03T15:21:05 IST" duration-ms="2538" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkBackToHome(java.lang.String,java.lang.String)[pri:11, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:21:05 IST" name="checkBackToHome" data-provider="fetchData" finished-at="2023-08-03T15:21:12 IST" duration-ms="6450" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkBackToHome -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:21:12 IST" name="tearDown" finished-at="2023-08-03T15:21:12 IST" duration-ms="255" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:21:12 IST" name="setUp" finished-at="2023-08-03T15:21:15 IST" duration-ms="2439" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkBackToHome(java.lang.String,java.lang.String)[pri:11, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:21:15 IST" name="checkBackToHome" data-provider="fetchData" finished-at="2023-08-03T15:21:19 IST" duration-ms="4220" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[ User handling the exception, couldnt reach the checkout page expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError:  User handling the exception, couldnt reach the checkout page expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at tests.CheckOutTest.checkBackToHome(CheckOutTest.java:184)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkBackToHome -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:21:19 IST" name="tearDown" finished-at="2023-08-03T15:21:19 IST" duration-ms="212" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:21:19 IST" name="setUp" finished-at="2023-08-03T15:21:22 IST" duration-ms="2472" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkBackToHome(java.lang.String,java.lang.String)[pri:11, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:21:22 IST" name="checkBackToHome" data-provider="fetchData" finished-at="2023-08-03T15:21:38 IST" duration-ms="16480" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[performance_glitch_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkBackToHome -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:21:38 IST" name="tearDown" finished-at="2023-08-03T15:21:38 IST" duration-ms="226" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:21:38 IST" name="setUp" finished-at="2023-08-03T15:21:41 IST" duration-ms="2301" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkBackToHome(java.lang.String,java.lang.String)[pri:11, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:21:41 IST" name="checkBackToHome" data-provider="fetchData" finished-at="2023-08-03T15:21:51 IST" duration-ms="10702" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[locked_out_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[ User handling the exception, couldnt reach the checkout page expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError:  User handling the exception, couldnt reach the checkout page expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at tests.CheckOutTest.checkBackToHome(CheckOutTest.java:184)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkBackToHome -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CheckOutTest@6e0f5f7f]" started-at="2023-08-03T15:21:52 IST" name="tearDown" finished-at="2023-08-03T15:21:52 IST" duration-ms="216" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.CheckOutTest -->
      <class name="tests.HomeTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.HomeTest@37d4349f]" started-at="2023-08-03T15:15:20 IST" name="setUp" finished-at="2023-08-03T15:15:22 IST" duration-ms="2323" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkLogOut(java.lang.String,java.lang.String)[pri:1, instance:tests.HomeTest@37d4349f]" started-at="2023-08-03T15:15:22 IST" name="checkLogOut" finished-at="2023-08-03T15:15:23 IST" duration-ms="1200" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkLogOut -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.HomeTest@37d4349f]" started-at="2023-08-03T15:15:23 IST" name="tearDown" finished-at="2023-08-03T15:15:23 IST" duration-ms="249" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.HomeTest -->
      <class name="tests.AddToCartTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:15:23 IST" name="setUp" finished-at="2023-08-03T15:15:26 IST" duration-ms="2345" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="addToCart(java.lang.String,java.lang.String)[pri:2, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:15:26 IST" name="addToCart" finished-at="2023-08-03T15:15:39 IST" duration-ms="12983" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addToCart -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:15:39 IST" name="tearDown" finished-at="2023-08-03T15:15:39 IST" duration-ms="263" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:15:39 IST" name="setUp" finished-at="2023-08-03T15:15:42 IST" duration-ms="2594" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="sortProducts(java.lang.String,java.lang.String)[pri:3, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:15:42 IST" name="sortProducts" data-provider="fetchData" finished-at="2023-08-03T15:15:42 IST" duration-ms="639" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sortProducts -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:15:42 IST" name="tearDown" finished-at="2023-08-03T15:15:42 IST" duration-ms="197" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:15:42 IST" name="setUp" finished-at="2023-08-03T15:15:45 IST" duration-ms="2526" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="sortProducts(java.lang.String,java.lang.String)[pri:3, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:15:45 IST" name="sortProducts" data-provider="fetchData" finished-at="2023-08-03T15:15:46 IST" duration-ms="890" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at tests.AddToCartTest.sortProducts(AddToCartTest.java:71)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sortProducts -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:15:46 IST" name="tearDown" finished-at="2023-08-03T15:15:46 IST" duration-ms="252" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:15:46 IST" name="setUp" finished-at="2023-08-03T15:15:49 IST" duration-ms="2417" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="sortProducts(java.lang.String,java.lang.String)[pri:3, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:15:49 IST" name="sortProducts" data-provider="fetchData" finished-at="2023-08-03T15:15:59 IST" duration-ms="10650" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[performance_glitch_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sortProducts -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:15:59 IST" name="tearDown" finished-at="2023-08-03T15:16:00 IST" duration-ms="235" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:16:00 IST" name="setUp" finished-at="2023-08-03T15:16:02 IST" duration-ms="2437" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="sortProducts(java.lang.String,java.lang.String)[pri:3, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:16:02 IST" name="sortProducts" data-provider="fetchData" finished-at="2023-08-03T15:16:03 IST" duration-ms="573" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[locked_out_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sortProducts -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:16:03 IST" name="tearDown" finished-at="2023-08-03T15:16:03 IST" duration-ms="206" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:16:03 IST" name="setUp" finished-at="2023-08-03T15:16:05 IST" duration-ms="2350" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="click_Highest_Product(java.lang.String,java.lang.String)[pri:4, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:16:05 IST" name="click_Highest_Product" data-provider="fetchData" finished-at="2023-08-03T15:16:11 IST" duration-ms="5760" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- click_Highest_Product -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:16:11 IST" name="tearDown" finished-at="2023-08-03T15:16:11 IST" duration-ms="268" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:16:11 IST" name="setUp" finished-at="2023-08-03T15:16:14 IST" duration-ms="2386" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="click_Highest_Product(java.lang.String,java.lang.String)[pri:4, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:16:14 IST" name="click_Highest_Product" data-provider="fetchData" finished-at="2023-08-03T15:16:30 IST" duration-ms="16085" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at tests.AddToCartTest.click_Highest_Product(AddToCartTest.java:111)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- click_Highest_Product -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:16:30 IST" name="tearDown" finished-at="2023-08-03T15:16:30 IST" duration-ms="243" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:16:30 IST" name="setUp" finished-at="2023-08-03T15:16:33 IST" duration-ms="2536" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="click_Highest_Product(java.lang.String,java.lang.String)[pri:4, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:16:33 IST" name="click_Highest_Product" data-provider="fetchData" finished-at="2023-08-03T15:16:49 IST" duration-ms="15816" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[performance_glitch_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- click_Highest_Product -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:16:49 IST" name="tearDown" finished-at="2023-08-03T15:16:49 IST" duration-ms="251" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:16:49 IST" name="setUp" finished-at="2023-08-03T15:16:51 IST" duration-ms="2296" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="click_Highest_Product(java.lang.String,java.lang.String)[pri:4, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:16:51 IST" name="click_Highest_Product" data-provider="fetchData" finished-at="2023-08-03T15:16:57 IST" duration-ms="5389" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[locked_out_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at tests.AddToCartTest.click_Highest_Product(AddToCartTest.java:111)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- click_Highest_Product -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:16:57 IST" name="tearDown" finished-at="2023-08-03T15:16:57 IST" duration-ms="209" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:16:57 IST" name="setUp" finished-at="2023-08-03T15:16:59 IST" duration-ms="2376" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifyProductDetails(java.lang.String,java.lang.String)[pri:5, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:16:59 IST" name="verifyProductDetails" data-provider="fetchData" finished-at="2023-08-03T15:17:05 IST" duration-ms="5564" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyProductDetails -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:05 IST" name="tearDown" finished-at="2023-08-03T15:17:05 IST" duration-ms="246" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:05 IST" name="setUp" finished-at="2023-08-03T15:17:08 IST" duration-ms="2418" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifyProductDetails(java.lang.String,java.lang.String)[pri:5, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:08 IST" name="verifyProductDetails" data-provider="fetchData" finished-at="2023-08-03T15:17:13 IST" duration-ms="5461" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyProductDetails -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:13 IST" name="tearDown" finished-at="2023-08-03T15:17:13 IST" duration-ms="229" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:13 IST" name="setUp" finished-at="2023-08-03T15:17:16 IST" duration-ms="2401" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifyProductDetails(java.lang.String,java.lang.String)[pri:5, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:16 IST" name="verifyProductDetails" data-provider="fetchData" finished-at="2023-08-03T15:17:26 IST" duration-ms="10435" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[performance_glitch_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyProductDetails -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:26 IST" name="tearDown" finished-at="2023-08-03T15:17:26 IST" duration-ms="221" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:26 IST" name="setUp" finished-at="2023-08-03T15:17:29 IST" duration-ms="2564" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifyProductDetails(java.lang.String,java.lang.String)[pri:5, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:29 IST" name="verifyProductDetails" data-provider="fetchData" finished-at="2023-08-03T15:17:34 IST" duration-ms="5378" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[locked_out_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at tests.AddToCartTest.verifyProductDetails(AddToCartTest.java:133)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyProductDetails -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:35 IST" name="tearDown" finished-at="2023-08-03T15:17:35 IST" duration-ms="212" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:35 IST" name="setUp" finished-at="2023-08-03T15:17:37 IST" duration-ms="2440" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkOut_Products_Validation(java.lang.String,java.lang.String)[pri:6, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:37 IST" name="checkOut_Products_Validation" data-provider="fetchData" finished-at="2023-08-03T15:17:38 IST" duration-ms="839" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkOut_Products_Validation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:38 IST" name="tearDown" finished-at="2023-08-03T15:17:38 IST" duration-ms="194" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:38 IST" name="setUp" finished-at="2023-08-03T15:17:41 IST" duration-ms="2449" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkOut_Products_Validation(java.lang.String,java.lang.String)[pri:6, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:41 IST" name="checkOut_Products_Validation" data-provider="fetchData" finished-at="2023-08-03T15:17:42 IST" duration-ms="903" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at tests.AddToCartTest.checkOut_Products_Validation(AddToCartTest.java:163)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkOut_Products_Validation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:42 IST" name="tearDown" finished-at="2023-08-03T15:17:42 IST" duration-ms="242" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:42 IST" name="setUp" finished-at="2023-08-03T15:17:45 IST" duration-ms="2460" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkOut_Products_Validation(java.lang.String,java.lang.String)[pri:6, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:45 IST" name="checkOut_Products_Validation" data-provider="fetchData" finished-at="2023-08-03T15:17:50 IST" duration-ms="5853" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[performance_glitch_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkOut_Products_Validation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:50 IST" name="tearDown" finished-at="2023-08-03T15:17:51 IST" duration-ms="214" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:51 IST" name="setUp" finished-at="2023-08-03T15:17:53 IST" duration-ms="2635" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkOut_Products_Validation(java.lang.String,java.lang.String)[pri:6, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:17:53 IST" name="checkOut_Products_Validation" data-provider="fetchData" finished-at="2023-08-03T15:18:04 IST" duration-ms="10795" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[locked_out_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkOut_Products_Validation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AddToCartTest@434a63ab]" started-at="2023-08-03T15:18:04 IST" name="tearDown" finished-at="2023-08-03T15:18:04 IST" duration-ms="211" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.AddToCartTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
